californium-core/src/main/java/org/eclipse/californium/core/network/stack/Block2BlockwiseStatus.java

```diff
@@ -189,7 +189,7 @@ public final synchronized boolean isNew(final Response response) {
 				// none observe exchanges are cleared on sending request
 				return false;
 			}
-			return orderer == null || orderer.isNew(response);
+			return order == null || order.isNew(response);
```

Comment:
Using the orderer, a new notify response updated the observe number, and that seems to cause issues.